.github/workflows/k8s-content.yaml

```diff
@@ -0,0 +1,168 @@
+---
+name: Kubernetes content
+
+on:
+  push:
+    branches: [ 'master' ]
+
+env:
+  COSIGN_EXPERIMENTAL: 1
+  IMAGE_NAME: ghcr.io/ComplianceAsCode/k8scontent
+
+jobs:
+  container:
+    runs-on: ubuntu-latest
+
+    permissions:
+      packages: write
+
+    outputs:
+      image-digest: ${{ steps.container_info.outputs.image-digest }}
+      image-tags: ${{ steps.container_info.outputs.image-tags }}
+
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+
+      - name: Login to ghcr.io
+        uses: docker/login-action@v1.14.1
+        with:
+          registry: ghcr.io
+          username: ${{ github.actor }}
+          password: ${{ secrets.GITHUB_TOKEN }}
+
+      # TODO: Rename ocp4_content to k8s_content and make it more general
+      - name: Build container image
+        run: |
+          revision="$(git rev-parse "${GITHUB_REF_NAME:-HEAD}")"
+          docker build \
+            -f ./Dockerfiles/ocp4_content \
+            -t "${IMAGE_NAME}:latest" \
+            -t "${IMAGE_NAME}:${GITHUB_REF_NAME}" \
+            -t "${IMAGE_NAME}:${revision}" \
+            --label "org.opencontainers.image.source=https://github.com/ComplianceAsCode/content" \
+            --label "org.opencontainers.image.created=$(date --iso-8601=seconds)" \
+            --label "org.opencontainers.image.title=k8scontent" \
+            --label "org.opencontainers.image.revision=${revision}" \
+            --label "org.opencontainers.image.version=${GITHUB_REF_NAME}" \
+            --label "org.opencontainers.image.licenses=MIT" \
+            --label "org.opencontainers.image.vendor=Equinix, Inc." \
```

Comment:
Can't we use something vendor agnostic here?

Reply:
oops! Copy pasta error. Sorry